src/lib/core/components/visualizations/InputVariables.tsx

```diff
@@ -16,14 +16,17 @@ import { Tooltip } from '@veupathdb/components/lib/components/widgets/Tooltip';
 import RadioButtonGroup from '@veupathdb/components/lib/components/widgets/RadioButtonGroup';
 import { isEqual } from 'lodash';
 
+// used for 2x2 axis variable style override
+import { twoBytwoInputStyle } from './implementations/MosaicVisualization';
+
 export interface InputSpec {
   name: string;
   label: string;
   /** Provide a string here to indicate that the input is readonly.
    * The string will be displayed instead of a variable selector.
    */
   readonlyValue?: string;
-  role?: 'axis' | 'stratification';
+  role?: 'axis' | 'stratification' | 'twoByTwoAxis';
```

Comment:
I'd like to find a way to avoid hard coding viz-specific values here. It might require refactoring a bit so that the consumer of `InputVariables` can specify `order` and `title`. Otherwise, we will likely end up with a lot of viz-specific stuff in here, as we continue to evolve the layout of various visualizations. What do you think @jernestmyers?